src/main/scala/fundamentals/level01/IntroExercises.scala

```diff
@@ -5,13 +5,19 @@ package fundamentals.level01
   * Level 1 focuses on basic Scala, including syntax and especially programming with functions.
   * We will go through a few exercises to familiarise ourselves with writing Scala.
   *
-  * What's a function?
-  *
-  * A function takes inputs and returns an output.
-  * It should always return the same output given the same inputs.
+  * An object:
+  * - An object is a class that has exactly one instance (a singleton). It is also a value.
```

Comment:
I think we can say: "An object is a class that has exactly one instance, i.e. a singleton." 

The "It is also a value" bit is a little confusing I think.